adafruit_as726x.py

```diff
@@ -527,20 +536,20 @@ class AS726x_UART(AS726x):
 
     """
 
-    def __init__(self, uart):
+    def __init__(self, uart: busio.UART):
```

Comment:
Just missing the ` -> None` return type here

Reply:
Hmm. Does __init__() return `None`, or does it return the instance of the class it's `init`'ing? In my own code I've always tended to leave the type annotation off of the `__init__` dunder method, but I'm not actually sure now what's correct here now that I think about it.

Reply:
I actually ran into this a few weeks ago when I ran into the `__new__()` method!  So `__new__()` returns a new instance of the classes and should be type as returning `"the_class"` (in quotes because you're typing with the class itself before done writing it, if that makes sense) whereas `__init__()` only initializes the class after creating the instance, so it doesn't actually return anything itself.

Reply:
Here's a DEV post about it:  https://dev.to/pila/constructors-in-python-init-vs-new-2f9j